MapboxNavigation/RouteVoiceController.swift

```diff
@@ -185,9 +183,9 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate, AVAudioP
 }
 
 @objc public protocol VoiceControllerDelegate {
-    @objc(voiceController:spokenInstrucionsDidFailWithError:routeProgress:)
-    optional func voiceController(_ voiceController: RouteVoiceController, spokenInstructionsDidFailWith error: Error, routeProgress: RouteProgress)
+    @objc(voiceController:spokenInstrucionsDidFailWithError:)
+    optional func voiceController(_ voiceController: RouteVoiceController, spokenInstructionsDidFailWith error: Error)
     
-    @objc(voiceController:didInterruptSpokenInstruction:withInstruction:routeProgress:)
-    optional func voiceController(_ voiceController: RouteVoiceController, didInterrupt: SpokenInstruction, with instruction: SpokenInstruction, routeProgress: RouteProgress)
+    @objc(voiceController:didInterruptSpokenInstruction:withInstruction:)
+    optional func voiceController(_ voiceController: RouteVoiceController, didInterrupt: SpokenInstruction, with instruction: SpokenInstruction)
```

Comment:
Incidentally, can we get some documentation on these public methods? Something like the comments you already have in ViewController.swift.